syntax = "proto3";

package Davidian.orbital;

message OrbitalElements{
    double semimajor_axis = 1;
    double eccentricity = 2;
    double longitude_asc_node = 3;
    double inclination = 4;
    double arg_periapsis = 5;
    double mean_anomaly_0 = 6;
}

message CartesianVector{
    double x = 1;
    double y = 2;
    double z = 3;
}